addons/spreadsheet_dashboard/static/src/bundle/grid_cursor/grid.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module */
+
+import { getNumberOfListFormulas } from "@spreadsheet/list/list_helpers";
+import spreadsheet from "@spreadsheet/o_spreadsheet/o_spreadsheet_extended";
+import { getNumberOfPivotFormulas } from "@spreadsheet/pivot/pivot_helpers";
+import { patch } from "@web/core/utils/patch";
+
+const { Grid } = spreadsheet.components;
+
+/* Used for t-key */
+let key = 0;
+
+patch(Grid.prototype, "spreadsheet_dashboard_grid_cursor", {
+    getCellClickableStyle(coordinates) {
+        return `
+      top: ${coordinates.top}px;
+      left: ${coordinates.left}px;
+      width: ${coordinates.width}px;
+      height: ${coordinates.height}px;
+    `;
+    },
+
+    getClickableCells() {
+        const cells = [];
+        const viewport = this.env.model.getters.getActiveViewport();
```

Comment:
This getter does not exist anymore tahnks to the  "freeze pane" ref. However, since you only need the delimiting zone to iterate on the cells, you can use the new getters `getSheetViewVisibleCols` and `getSheetViewVisibleRows` which will do the trick just fine :)